modules/custom/social_queue_storage/social_queue_storage.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the social_queue_storage module.
+ */
+
+use Drupal\Core\Field\BaseFieldDefinition;
+
+/**
+ * Add finished field to the "queue_storage_entity" entity.
+ */
+function social_queue_storage_update_8001(&$sandbox) {
```

Comment:
Don't we want this as an install hook considering this is new? 🤔

Reply:
This was mainly there for @bramtenhove.
It could indeed be removed.